packages/backend/package.json

```diff
@@ -10,7 +10,6 @@
 		"build": "tsc -p tsconfig.build.json",
 		"prepublish": "npm run build",
 		"version": "git add -A src",
-		"postversion": "git push && git push --tags",
```

Comment:
Why is it removed?

Reply:
It's aleftover after the times when waggle was a separate package.